src/Polly/Polly.csproj

```diff
@@ -1,12 +1,12 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
These changes should be reverted, they're made as part of the build itself based on the Git versioning and shouldn't be committed.